doc/manpages/conf.py

```diff
@@ -323,6 +323,7 @@
     ('nhrpd', 'nhrpd', fwfrr.format("a Next Hop Routing Protocol "), [], 8),
     ('pimd', 'pimd', fwfrr.format("a PIM "), [], 8),
     ('pbrd', 'pbrd', fwfrr.format("a PBR "), [], 8),
+    ('sharpd', 'sharpd', fwfrr.format("an SHARP "), [], 8),
```

Comment:
as nitpicky as this is -- I think this is a typo, should be `a SHARP`

Reply:
done